Relax tolerance for the following with Intel oneAPI: TestBatch.test_solve and TestBatch.test_lu_solve
Patch by Simon Branford (University of Birmingham)
--- scipy/linalg/tests/test_batch.py.orig	2025-09-09 14:48:42.485545000 +0100
+++ scipy/linalg/tests/test_batch.py	2025-09-09 14:48:54.003644602 +0100
@@ -445,7 +445,7 @@
         if len(bdim) == 1:
             x = x[..., np.newaxis]
             b = b[..., np.newaxis]
-        assert_allclose(A @ x - b, 0, atol=1.5e-6)
+        assert_allclose(A @ x - b, 0, atol=1.7e-6)
         assert_allclose(x, np.linalg.solve(A, b), atol=3e-6)
 
     @pytest.mark.parametrize('bdim', [(5,), (5, 4), (2, 3, 5, 4)])
@@ -459,7 +459,7 @@
         if len(bdim) == 1:
             x = x[..., np.newaxis]
             b = b[..., np.newaxis]
-        assert_allclose(A @ x - b, 0, atol=1.5e-6)
+        assert_allclose(A @ x - b, 0, atol=1.7e-6)
         assert_allclose(x, np.linalg.solve(A, b), atol=3e-6)
 
     @pytest.mark.parametrize('l_and_u', [(1, 1), ([2, 1, 0], [0, 1 , 2])])
